keyboards/flx/virgo/virgo.h

```diff
@@ -17,27 +18,25 @@
 
 #include "quantum.h"
 
-#define LAYOUT( \
-    K00,   K01, K11, K02, K12,    K03,           K13, K04, K14,   K05, K15, K06, K16,     K07, K17, K08,  \
-    K20, K30, K21, K31, K22, K32, K23,           K33, K24, K34, K25, K35, K26, K36, K66,  K27, K37, K28,  \
-    K40,   K50, K41, K51, K42, K52,           K43, K53, K44, K54, K45, K55, K46, K56,     K47, K57, K48,  \
-    K60,    K70, K61, K71, K62, K72,           K63, K73, K64, K74, K65, K75, K76,                         \
-    K80,  K90, K81, K91, K82, K92, K83,      KA3, K93, K84, K94, K85, K95, K86,     K96,       K97,       \
-    KA0,        KB0,    KA1, KB2,            KB4,          KB5, KA6,        KB6,          KA7, KB7, KA8   \
+#define LAYOUT_all( \
```

Comment:
Why did you do this? 

`K30` means the key connected to row 3, column 0. That is why it is in that format.

Reply:
Personal ease of use and layout visualisation. The new is functional, is it wrong?

Reply:
The old is functional too and adheres with what we've been teaching people.